### PR TITLE
Create button component.

### LINKED ISSUE #3: Create Button component

### PR DESCRIPTION
**Pull request by laurandarius**
Closes #3

### CODE REVIEW

**Comment by laurandarius** [comment]
@lgnstn Please review my PR.

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styled from "styled-components";
+
+const DisplayLayout = styled.div`
```
Comment (lgnstn) [inline src/components/Display.js]: This component shouldn't be changed in this PR. Sorry for the misunderstanding :)

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React, { useState } from "react";
+
+const Button = props => (
```
Comment (lgnstn) [inline src/components/Button.js:17]: Add style for this component in this PR.

**Review by lgnstn — CHANGES_REQUESTED** [review]

**Comment by laurandarius** [comment]
@lgnstn Please review my PR.

File: src/components/Button.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+const ButtonLayout = styled.button`
+  width: 58.2px;
+  height: 47.5px;
+  background-color: gainsboro;
+  font-size: 22px;
+  color: black;
+`;
+
+const Button = props => <ButtonLayout>{props.value}</ButtonLayout>;
```
Comment (lgnstn) [inline src/components/Button.js]: Why did you remove the possibility to configure the size and the color?

**Review by lgnstn — CHANGES_REQUESTED** [review]

**Review by lgnstn — APPROVED** [review]

**Pull request merged**